src/store/kanbanStore.ts

```
@@ -1,423 +1,423 @@
+
+// Firestoreは undefined 値をサポートしていないため、除去する
+// nullの場合はdeleteField()を使用してフィールドを削除する
+function removeUndefinedFields<T extends Record<string, unknown>>(obj: T, forFirestore = false): Partial<T> {
+    const result: Partial<T> = {}
+    for (const key in obj) {
+        if (obj[key] === undefined) {
+            // undefinedは除去
+            continue
+        } else if (obj[key] === null && forFirestore) {
+            // Firestoreの場合、nullはdeleteField()に変換してフィールドを削除
+            ;(result as Record<string, unknown>)[key] = deleteField()
```

ESLint

[error] Unnecessary semicolon.

(no-extra-semi)